public/app/features/search/types.ts

```diff
@@ -80,6 +80,7 @@ export interface DashboardViewItem {
   // For enterprise sort options
   sortMeta?: number | string; // value sorted by
   sortMetaName?: string; // name of the value being sorted e.g. 'Views'
+  repository?: string;
```

Comment:
^^ required?

Reply:
We can make it required after it comes form the API, otherwise it'll break a lot of types on FE.